hw/ip/otbn/doc/dv/index.md

```diff
@@ -439,34 +461,39 @@ This instruction uses the `bnaq` encoding schema, with covergroup `enc_bnaq_cg`.
 This instruction uses the `bnaq` encoding schema, with an extra field not present in `bn.mulqacc`.
 Encoding-level coverpoints are tracked in covergroup `enc_bnaqw_cg`.
 
-Specific points as for `BN.MULQACC`, plus the generic flag group cover points.
+- Cross `wrs1_qwsel` with `wrs2_qwsel` to make sure they are applied to the right inputs
+- See the accumulator overflow
 
 #### BN.MULQACC.SO
 
 This instruction uses the `bnaq` encoding schema, with an extra field not present in `bn.mulqacc`.
 Encoding-level coverpoints are tracked in covergroup `enc_bnaqs_cg`.
 
-Specific points as for `BN.MULQACC` plus the following:
-
+- Cross `wrs1_qwsel` with `wrs2_qwsel` to make sure they are applied to the right inputs
+- See the accumulator overflow
 - Cross the generic flag updates with `wrd_hwsel`, since the flag changes are different in the two modes.
 
 #### BN.SUB
 
 This instruction uses the `bnaf` encoding schema, with covergroup `enc_bna_cg`.
 
-Specific points as for `BN.ADD`.
+- Extremal values of shift for both directions where the shifted value is nonzero
```

Comment:
Mention of shift seems out of place.

Reply:
As with `bn.add`, [`bn.sub`](https://docs.opentitan.org/hw/ip/otbn/doc/isa/#bnsub) has a shift that's applied to one of its arguments.